controllers/operands/kubevirt.go

```diff
@@ -680,12 +681,24 @@ func getKVSeccompConfig() *kubevirtcorev1.SeccompConfiguration {
 	}
 }
 
+func getKubevirtCRAnnotations(hc *hcov1beta1.HyperConverged) map[string]string {
+	var annotations map[string]string
+
+	if hc.Spec.FeatureGates.AlignCPUs != nil && *hc.Spec.FeatureGates.AlignCPUs {
+		annotations = make(map[string]string)
+		annotations["alpha.kubevirt.io/EmulatorThreadCompleteToEvenParity"] = ""
+	}
+
+	return annotations
+}
```

Comment:
Not sure about this function. It's ok for create KV, but for the upgrade usecase we should be more careful, not to delete user annotations. We'll need to review the `func (*kubevirtHooks) updateCr()` method to make sure we merge it to the existing CR.

Reply:
That's a great point! Adding the necessary logic to the UpdateCR flow.
DONE